app/messaging/process-debt-data-message.js

```diff
@@ -1,6 +1,10 @@
+const processPaymentRequest = require('../inbound/payment-request')
```

Comment:
This needs to be wrapped in curly brackets or it will throw an error that it isn't a function

`const { processPaymentRequest } = require('../inbound/payment-request')`

Reply:
Code file has been updated. New unit-test-file that would have detected the issue added.